yt_dlp/extractor/youtube.py

```diff
@@ -654,18 +642,23 @@ def _extract_badges(self, renderer: dict):
         return badges
 
     @staticmethod
-    def _join_text_entries(runs):
-        text = None
-        for run in runs:
-            if not isinstance(run, dict):
-                continue
-            sub_text = try_get(run, lambda x: x['text'], compat_str)
-            if sub_text:
-                if not text:
-                    text = sub_text
-                    continue
-                text += sub_text
-        return text
+    def _get_text(data, getter=None, max_runs=None):
+        for get in variadic(getter):
+            d = try_get(data, get) or data
```

Comment:
```suggestion
            d = try_get(data, get) if get is not None else data
```
right? since it shouldn't fallback to data if try_get returns None